code/src/types/order.types.ts

```diff
@@ -0,0 +1,36 @@
+import { Pageable, Sort } from "@/types/common";
+import { orderStatuses } from "@/constants/orderStatuses";
+import { Product } from "@/types/product.types";
+import { PostAddress, Receiver } from "@/types/delivery.types";
+
+export type OrderStatuses = keyof typeof orderStatuses;
```

Comment:
Use singular form when declaring types. It is recommended by tepescript experts. For example compare following:

```jsx
type Orders = {...}

const orders: Orders[] = []
```

or

```jsx
type Order = {...}

const orders: Order[] = []
```

As you can see, the second approach is basically butter oil.

Reply:
Thank you for your advice. Fixed